lib/matplotlib/ticker.py

```diff
@@ -800,14 +800,27 @@ class LogFormatter(Formatter):
     """
     Format values for log axis.
     """
-    def __init__(self, base=10.0, labelOnlyBase=True):
+    def __init__(self, base=10.0, labelOnlyBase=False,
+                 sublabel_filtering=False):
         """
         `base` is used to locate the decade tick, which will be the only
         one to be labeled if `labelOnlyBase` is ``True``.
+
+        Parameter
+        ---------
+        base : float, optional, default: 10.
+            base of the logarithm.
+
+        labelOnlyBase : bool, optional, default: False
+             whether to only label decades ticks.
```

Comment:
* The descriptions starts by a 5-space indent, doesn't it?

Sidenote: the last parameter has a description starting with a capital letter while the first two parameters haven't.